e2e-pw/src/oss/specs/smoke-tests/embeddings.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { test as base } from "src/oss/fixtures";
+import { GridPom } from "src/oss/poms/grid";
+import { PanelPom } from "src/oss/poms/panels/panel";
+import { EmbeddingsPom } from "src/oss/poms/panels/embeddings-panel";
+import { getUniqueDatasetNameWithPrefix } from "src/oss/utils";
+
+const datasetName = getUniqueDatasetNameWithPrefix("smoke-quickstart");
+
+const test = base.extend<{ grid: GridPom; panel: PanelPom }>({
```

Comment:
`embeddings` is missing in the type list
```suggestion
const test = base.extend<{ embeddings: EmbeddingsPom, grid: GridPom; panel: PanelPom }>({
```